src/Opdex.Platform.Common/Enums/TransactionEventType.cs

```diff
@@ -47,5 +47,9 @@ public enum TransactionEventType
     VaultProposalPledgeEvent = 28,
     VaultProposalWithdrawPledgeEvent = 29,
     VaultProposalVoteEvent = 30,
-    VaultProposalWithdrawVoteEvent = 31
+    VaultProposalWithdrawVoteEvent = 31,
+
+    // Interflux tokens
+    OwnershipTransferredEvent = 32,
+    SupplyChangeEvent = 33,
```

Comment:
Side note, maybe you could add to this PR. 

These enums are off by 1 from the `TransactionLogType` because a `ReservesEvent` doesn't exist in the Liquidity Pools section. Would need a response model and probably a DTO assembler so can be done as a separate task too, up to you.

Reply:
Happy to do this - I think we'll need to rewind testnet back to the first block if we want the indexer to pick up these events

Reply:
Appears to have resolved the issue with missing `sortOrder` values